lib/lamby/tasks.rake

```diff
@@ -0,0 +1,7 @@
+namespace :lamby do
+  task :proxy_server => [:environment] do
+    require 'webrick'
```

Comment:
Yes, there is a peer (not runtime) dep on WEBrick here. If this feature were more public, I would document that somewhere. Will likely add that docs to the live development project.